.prettierignore

```diff
@@ -1,3 +1,6 @@
+.eslintignore
+.prettierignore
```

Comment:
Why should prettier ignore these two files?

Reply:
these changes are meanwhile already on `develop`

Reply:
yes, I took them from the other calendar fix